core/src/test/scala/unit/kafka/integration/KafkaServerTestHarness.scala

```diff
@@ -54,11 +58,18 @@ trait KafkaServerTestHarness extends ZooKeeperTestHarness {
 
   protected def securityProtocol: SecurityProtocol = SecurityProtocol.PLAINTEXT
   protected def trustStoreFile: Option[File] = None
+  def clusterAclArgs: Array[String] = Array("--authorizer-properties", 
+                                            s"zookeeper.connect=$zkConnect",
+                                            s"--add",
+                                            s"--cluster",
+                                            s"--allow-principal=$kafkaPrincipal")
```

Comment:
Actually, just initializing with empty strings didn't work, it caused a bunch of test cases to fail. I've actually came up with a solution that I like better. I've added a `setClusterAcl` method that is a no-op by default, but needs to be overriden if the ACL is supposed to be set. This way we avoid the awkward if that was there before to determine whether to set the ACL or not.